viite-UI/src/model/RoadCollection.js

```diff
@@ -95,32 +111,21 @@
             _.contains(selectedIds, roadLink.getId());
           });
         }).concat(getSelectedRoadLinks());
+
+        historicRoadLinks = _.filter(roadLinkGroups, function(group) {
+          groupDataSourceFilter(group, LinkSource.HistoryLinkInterface);
```

Comment:
LinkSource relates to VVH historic links, not historic road addresses of road links -> should not be filtered at client side at all

Reply:
Actually unrelated to the fact.